app_grim_reefer/boards/shields/mikroe_click/mikroe_click.overlay

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Wipe shields. Already defined.

Reply:
fixed in [6cf8032](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/6cf803229d38690de83d6db332228645b0495534)